sdk/jdbc/azure-identity-extensions/README.md

```diff
@@ -1,4 +1,4 @@
-- [Azure identity authentication providers plugin library for Java](#azure-identity-authentication-providers-plugin-library-for-java)
+- [Azure identity authentication extensions library for Java](#azure-identity-authentication-extensions-library-for-java)
```

Comment:
The header only supports patterns:`* Azure .+ (client|plugin|shared) library for Java`